source/backend/routes/users.js

```diff
@@ -86,4 +88,45 @@ router.delete('/:savedRecipeId', async (req, res) => {
   }
 });
 
+/* POST /auth/signup */
+/* Creates new user in database with username, password and email -> unique user/email 
+required */
+router.post('/auth/signup', async (req, res) => {
+  try{
+    const { username } = req.username;
+    const { password } = req.password;
```

Comment:
I think this might need to be changed to 
```
const { username } = req.body.user
const { password } = req.body.user

// if you want to get fancy
// const {username, password} = req.body.user
```

Reply:
This can just be `const {username, password} = req.body` doesn't make sense to add an extra user tag in the req.body object.